build/charts/theia/README.md

```diff
@@ -0,0 +1,46 @@
+# theia
+
+![Version: 0.1.0-dev](https://img.shields.io/badge/Version-0.1.0--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0-dev](https://img.shields.io/badge/AppVersion-0.1.0--dev-informational?style=flat-square)
+
+Antrea Network Flow Visibility
+
+**Homepage:** <https://antrea.io/>
+
+## Source Code
+
+* <https://github.com/antrea-io/theia>
+
+## Requirements
+
+Kubernetes: `>= 1.16.0-0`
+
+## Values
```

Comment:
to decrease scrolling right, shall we switch the Default and Description column? like https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md#configuration

Reply:
This doc is auto-generated by [helm-docs](https://github.com/norwoodj/helm-docs/releases/v1.7.0) as Antrea side does. I do not find a way switch the column. I can revisit the problem if the doc generation way is updated in the future.